### PR TITLE
various onboarding fixed

### PR DESCRIPTION
**Pull request by IAmMorrow**
fixed device specific wording
added link to recovery phrase help
last question of the quizz now show "finish" button

### CODE REVIEW

**Comment by machard** [comment]
You will update the first screen illustration as well?

**Comment by cthepot-ledger** [comment]
- 4185 requires bold font inside drawer on "never" and "always"
- 4177 is done
- 4181 is done
- 4127 is done

@IAmMorrow

**Comment by cthepot-ledger** [comment]
can we process 4247 at the same time please

**Comment by valpinkman** [comment]
Fixed LL-4247 as well

image

**Review by juan-cortes — CHANGES_REQUESTED** [review]

File: src/renderer/components/Onboarding/Screens/SelectDevice/devices.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const devices = {
```
[the remark this answers is not in the document]
Comment (juan-cortes) [reply src/renderer/components/Onboarding/Screens/SelectDevice/devices.js:1]: To me, this design change means we should reflect this "shorthand" product name in live-common.
For now we can have it locally, but maybe worth taking a look when @gre is back

**Review by juan-cortes — DISMISSED** [review]
lgtm

**Comment by machard** [comment]
can you update the snapshot ?

File: src/renderer/components/Onboarding/Help/RecoverySeed.js [hunk 2]
```diff
@@ -86,15 +89,17 @@ export function RecoverySeed() {
...
-        <Text
-          mt="8px"
-          color="palette.text.shade100"
-          ff="Inter|Regular"
-          fontSize="14px"
-          lineHeight="19.5px"
-        >
-          {t("onboarding.drawers.recoverySeed.link")}
-        </Text>
+        <FakeLink onClick={() => openURL(urls.whatIsARecoveryPhrase)}>
```
Comment (valpinkman) [inline src/renderer/components/Onboarding/Help/RecoverySeed.js:92]: inside useCallback

File: src/renderer/modals/Terms/index.js [hunk 3]
```diff
@@ -94,20 +95,17 @@ const TermsModal = ({ showClose = false }: { showClose?: boolean }) => {
...
-                    ff="Inter|SemiBold"
-                    color="alertRed"
-                    fontSize={4}
-                    style={{ marginLeft: 8, flex: 1 }}
-                  >
-                    <Trans i18nKey="Terms.switchLabelLoss" />
+                  <Text ff="Inter|SemiBold" fontSize={4} style={{ marginLeft: 8, flex: 1 }}>
+                    <Trans i18nKey="Terms.switchLabelTerms">
+                      <Text onClick={onSwitchAccept} />
+                      <Text onClick={() => openURL(urls.terms)} color="wallet" />
```
Comment (valpinkman) [inline src/renderer/modals/Terms/index.js:107]: inside useCallback

File: src/renderer/modals/Terms/index.js [hunk 4]
```diff
@@ -116,24 +114,24 @@ const TermsModal = ({ showClose = false }: { showClose?: boolean }) => {
...
+                    isChecked={acceptedPrivacyPolicy}
+                    onChange={onSwitchAcceptPrivacyPolicy}
+                    id="modal-terms-privacy-policy"
                   />
                   <Text ff="Inter|SemiBold" fontSize={4} style={{ marginLeft: 8, flex: 1 }}>
-                    <Trans i18nKey="Terms.switchLabel">
-                      <Text onClick={() => openURL(url)} color="wallet" />
+                    <Trans i18nKey="Terms.switchLabelPrivacyPolicy">
+                      <Text onClick={onSwitchAcceptPrivacyPolicy} />
+                      <Text onClick={() => openURL(urls.privacyPolicy)} color="wallet" />
```
Comment (valpinkman) [inline src/renderer/modals/Terms/index.js:126]: inside useCallback

**Review by valpinkman — CHANGES_REQUESTED** [review]

**Comment by gre** [comment]
CI to check

**Comment by machard** [comment]
@IAmMorrow I think onboarding test fail because #modal-terms-checkbox-loss is gone and firmware should pass when you rerun, its an issue most likely due to a slow be

**Review by Arnaud97234 — CHANGES_REQUESTED** [review]
On onboarding -> Genuine check Success screen the device model should be be corresponding to the device selected at the beginning of the Onboarding. It's always "Your Ledger Nano X is genuine ..." even when selecting a Blue or a NanoS.
Screenshot 2021-01-13 at 12 38 45

**Review by Arnaud97234 — DISMISSED** [review]

**Comment by gre** [comment]
@Arnaud97234  is this mergeable? I'm confused by the :x: and then ✅

**Review by nabil-brn — CHANGES_REQUESTED** [review]
image

It's missing spaces before and after device icons.
---
2.20.0-sha.ad97b5a17864f73614b430a7101ad5ca69942d79-win.exe.zip

**Comment by nabil-brn** [comment]
image

Last answer is overlapping the illustration.

---
2.20.0-sha.ad97b5a17864f73614b430a7101ad5ca69942d79-win.exe.zip

**Comment by gre** [comment]
@IAmMorrow can you check? thanks

**Comment by machard** [comment]
Capture d’écran 2021-01-22 à 09 40 41

Some ids have been removed that are necessary for the onboarding test flow

**Review by valpinkman — APPROVED** [review]

**Review by nabil-brn — CHANGES_REQUESTED** [review]
https://github.com/LedgerHQ/ledger-live-desktop/pull/3399#pullrequestreview-569056818

**Comment by gre** [comment]
enjoy your incoming screenshot reset :trollface:

**Comment by gre** [comment]
:itshappening:

**Review by gre — APPROVED** [review]

**Review by nabil-brn — APPROVED** [review]
✓ QA-ed

---
2.20.0-sha.181927b52f5e6f10ff48864190e2e944d6698c2b-mac.dmg

**Pull request merged**